sdk/src/main/java/com/uid2/sdk/network/DefaultNetworkSession.kt

```diff
@@ -0,0 +1,75 @@
+package com.uid2.sdk.network
+
+import com.uid2.sdk.extensions.decodeJsonToMap
+import java.net.HttpURLConnection
+import java.net.URL
+import java.net.URLEncoder
+
+/**
+ * A default implementation of NetworkSession that leverages HttpUrlConnection to make the necessary
+ * GET and POST requests.
+ */
+open class DefaultNetworkSession : NetworkSession {
+
+    override fun loadData(url: URL, request: NetworkRequest): Map<String, Any> {
+        val connection = openConnection(url).apply {
+            requestMethod = request.type.toRequestMethod()
+
+            setRequestProperty("Accept", "application/json")
```

Comment:
We'll need to add Client Version and Content Type headers -->

https://github.com/IABTechLab/uid2-ios-sdk/blob/fbbfdbdc6c5bf487cfafe5a121e4d73e1628d15e/Sources/UID2/UID2Client.swift#L33-L34

Reply:
Yup, these headers will also be added within the Android UID2Client class.